src/main/java/frc/robot/Robot.java

```diff
@@ -23,6 +28,8 @@
 
   private final Intake intake = new Intake(map.getIntakeMap());
 
+  private Shooter shooter = new Shooter(map.getShooterMap()); // gets shootermap and makes a shooter subsystem
```

Comment:
Comment isn't really necessary here - comments should generally explain why you're doing something this way